src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -274,11 +272,6 @@
                                 x:Uid="General_SettingsBackupAndRestoreStatusInfo"
                                 HorizontalContentAlignment="Left"
                                 ContentAlignment="Vertical">
-                                <labs:SettingsCard.Resources>
```

Comment:
While updating this, we might want to adopt:

![image](https://user-images.githubusercontent.com/9866362/230791811-e66b6b3d-594e-41d7-8b79-c48925e2b176.png)


i.e. more spacing between key - value. Dropping the ":". Or we keep it as is and update in a separate PR?

Reply:
Good idea. Let's do that in another pr.

Reply:
Yet, in a different area in Win Settings
![image](https://user-images.githubusercontent.com/65828559/230795067-5b6fbdb0-ce40-4f17-923d-f8f228430e9f.png)

Reply:
I'd set it to auto and set a fixed ColumnSpacing. That should always give the correct spacing :)?